spec/computable/registry/claim-reward.spec.ts

```diff
@@ -0,0 +1,129 @@
+import * as ganache from 'ganache-cli'
+import Web3 from 'web3'
+import { Contract, Block } from 'web3/types.d'
+import { stringToBytes, increaseTime } from '../../helpers'
+import Erc20 from '../../../src/contracts/erc-20'
+import Voting from '../../../src/contracts/plcr-voting'
+import Parameterizer from '../../../src/contracts/parameterizer'
+import Registry from '../../../src/contracts/registry'
+import { ParameterDefaults, NAME } from '../../../src/constants'
+import {
+  deployDll,
+  deployAttributeStore,
+  maybeParseInt,
+  eventReturnValues,
+} from '../../../src/helpers'
+
+const provider:any = ganache.provider(),
+  web3 = new Web3(provider)
+
+let erc20:Erc20,
+  dll:Contract,
+  store:Contract,
+  voting:Voting,
+  parameterizer:Parameterizer,
+  registry:Registry,
+  owner:string,
+  applicant:string,
+  challenger:string,
+  voter:string,
+  proposer:string
+
+describe('Registry: Claim Reward', () => {
+  beforeEach(async () => {
+    [owner, applicant, challenger, voter, proposer] = await web3.eth.getAccounts()
+
+    erc20 = new Erc20(owner)
+    const tokenAddress = await erc20.deploy(web3)
+    erc20.setProvider(provider)
+
+    dll = await deployDll(web3, owner)
+    dll.setProvider(provider)
+    const dllAddress = dll.options.address
+
+    store = await deployAttributeStore(web3, owner)
+    store.setProvider(provider)
+    const attributeStoreAddress = store.options.address
+
+    voting = new Voting(owner)
+    const votingAddress = await voting.deploy(web3, { tokenAddress, dllAddress, attributeStoreAddress })
+    voting.setProvider(provider)
+
+    parameterizer = new Parameterizer(owner)
+    const parameterizerAddress = await parameterizer.deploy(web3, { tokenAddress, votingAddress })
+    parameterizer.setProvider(provider)
+
+    registry = new Registry(owner)
+    const registryAddress = await registry.deploy(web3, { tokenAddress, votingAddress, parameterizerAddress, name: NAME })
+    registry.setProvider(provider)
+
+    //owner approves voting and reg to spend
+    // await erc20.approve(votingAddress, 1000000)
```

Comment:
remove the comments here , looks like voting is approved further down

Reply:
Done